### PR TITLE
[DONE] Feature import video - Manage BBB recording v2

### PR DESCRIPTION
**Pull request by LoicBonavent**
Add management for BBB recordings in the import_video module :
 - for BBB external recording where there is no available video file (only a presentation file), the option "Import to Pod as a video file" is not allowed.
 - added recording state to explain this.

### CODE REVIEW

**Review by ptitloup — CHANGES_REQUESTED** [review]
retirer l'exception flake8 et refaire le make lang / make compile

File: pod/import_video/views.py [hunk 1]
```diff
@@ -154,7 +153,7 @@ def upload_external_recording_to_pod(request, record_id):
 
 
 @login_required(redirect_field_name="referrer")
-def external_recordings(request):
+def external_recordings(request):  # noqa: C901
```
[the remark this answers is not in the document]
Comment (LoicBonavent) [reply pod/import_video/views.py]: - Noqa : C901 supprimé pour la fonction external_recordings
 - Ajout de toutes les traductions après un make lang et un make compilelang

**Review by ptitloup — APPROVED** [review]
Ok pour moi, merci !

**Review by vsabatie — APPROVED** [review]

File: pod/import_video/utils.py [hunk 2]
```diff
@@ -195,6 +195,22 @@ def save_video(request, dest_file, recording_name, description, date_evt=None):
...
+        source_url (String): Source URL
+
+    Returns:
+        Boolean: file exists (True) or not (False)
+    """
+    response = requests.head(source_url)
+    if response.status_code < 400:
+        return True
+    else:
+        return False
```
Comment (mattbild) [inline pod/import_video/utils.py:211]: return requests.head(source_url).status_code < 400

File: pod/import_video/views.py [hunk 3]
```diff
@@ -589,9 +555,9 @@ def upload_youtube_recording_to_pod(request, record_id):
             "Try changing the access rights to the video directly in Youtube."
         )
         raise ValueError(msg)
-    except PytubeError:
+    except PytubeError as pterror:
         msg = {}
-        msg["error"] = _("YouTube error")
+        msg["error"] = _("YouTube error '%s'" % (mark_safe(pterror)))
```
Comment (mattbild) [inline pod/import_video/views.py:560]: msg = {"error": _("YouTube error '%s'" % (mark_safe(pterror))), 
           "message": _(
        "YouTube content is inaccessible. "
        "This content does not appear to be publicly available."
    )}

**Pull request merged**